docker/README.md

```diff
@@ -0,0 +1,45 @@
+# S3 to webDAV Proxy
```

Comment:
Better place for this README file? replace the root README?

Reply:
yes, maybe replace the root README for now, mention that it is based on `rclone` and that there is the plan to bring this back into the upstream repo